client/state/atomic/transfers/actions.ts

```diff
@@ -0,0 +1,55 @@
+import {
+	ATOMIC_PLUGIN_INSTALL_INITIATE_WITH_TRANSFER,
+	ATOMIC_PLUGIN_INSTALL_REQUEST_TRANSFER_STATUS,
+	ATOMIC_PLUGIN_INSTALL_SET_TRANSFER_STATUS,
+} from 'calypso/state/action-types';
+import 'calypso/state/data-layer/wpcom/sites/atomic/transfers';
+import 'calypso/state/data-layer/wpcom/sites/atomic/transfers/latest';
+import 'calypso/state/atomic/init';
+
+/**
+ * Initiate Atomic transfer with plugin install.
+ *
+ * @param {string} siteId Site ID.
+ * @param {string} softwareSet Software set slug.
+ * @returns {object} An action object.
+ */
+export const initiateAtomicTransferWithPluginInstall = (
+	siteId: number,
+	softwareSet: string
+) => ( {
+	type: ATOMIC_PLUGIN_INSTALL_INITIATE_WITH_TRANSFER,
+	siteId,
+	softwareSet,
+	meta: {
+		dataLayer: {
+			trackRequest: true,
```

Comment:
We should be able to make use of this feature to track the request lifecycle in our component (e.g. isLoading checks)